health/health.h

```diff
@@ -90,4 +90,6 @@ extern void health_label_log_save(RRDHOST *host);
 extern char *health_edit_command_from_source(const char *source);
 extern void sql_refresh_hashes(void);
 
+extern void health_add_host_labels(void);
```

Comment:
@underhood I removed `extern` from all headers for functions. Do you mind to update the  PR?